lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -810,3 +823,122 @@ PYI_POST_OFFICE:
     end:
       targetJourney: INELIGIBLE
       targetState: INELIGIBLE
+
+# Mitigation journey (03) same session - invalid-dl
+MITIGATION_03_OPTIONS:
+  response:
+    type: page
+    pageId: pyi-driving-licence-no-match-another-way
+    mitigationStart: invalid-dl
+  events:
+    next:
+      targetState: MITIGATION_PP_CRI_UK_PASSPORT
+    end:
+      targetJourney: INELIGIBLE
+      targetState: INELIGIBLE
```

Comment:
I've just heard back about the UCD, and we should be using `targetState: INELIGIBLE_SKIP_MESSAGE` here. That will take the user straight back to the service without showing them an extra "another way" page.

Reply:
done...return to RP instead of another way.